codalab/scripts/initialize_teams.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# Run this script from the CodaLab virtual environment to insert
+# initial data required by the web app into the database.
+
```

Comment:
Hmm, but it looks the same as `initialize.py`? Is it different, meant to be run over and over?